config/webpack.common.js

```diff
@@ -102,6 +102,7 @@ module.exports = {
       "window.Tether": "tether"
     })
   ],
+  postcss: () => [autoprefixer({browsers: ['last 2 versions']})],
```

Comment:
what is the purpose of specifying the last 2 versions of major browsers? i couldn't find a good explanation in the docs.